packages/material-ui/src/Breadcrumbs/BreadcrumbCollapsed.test.js

```diff
@@ -13,35 +13,25 @@ describe('<BreadcrumbCollapsed />', () => {
   });
 
   it('should render an icon', () => {
-    const { container } = render(<BreadcrumbCollapsed />);
+    const { container, getByRole } = render(<BreadcrumbCollapsed />);
 
     expect(container.querySelectorAll('svg').length).to.equal(1);
-    expect(container.firstChild).to.have.class(classes.root);
+    expect(getByRole('button')).to.have.class(classes.button);
   });
 
-  describe('prop: onKeyDown', () => {
-    it(`should be called on key down - Enter`, () => {
-      const handleClick = spy();
-      const { container } = render(<BreadcrumbCollapsed onClick={handleClick} />);
-      const expand = container.firstChild;
-
-      act(() => {
-        expand.focus();
-        fireEvent.keyDown(expand, { key: 'Enter' });
-      });
+  it('renders a native <button>', () => {
+    const { getByRole } = render(<BreadcrumbCollapsed />);
 
-      expect(handleClick.callCount).to.equal(1);
-    });
+    expect(getByRole('button')).to.have.property('nodeName', 'BUTTON');
+  });
 
-    it(`should be called on key up - Space`, () => {
+  describe('prop: onClick', () => {
+    it(`should be called when clicked`, () => {
       const handleClick = spy();
-      const { container } = render(<BreadcrumbCollapsed onClick={handleClick} />);
-      const expand = container.firstChild;
+      const { getByRole } = render(<BreadcrumbCollapsed onClick={handleClick} />);
+      const expand = getByRole('button');
 
-      act(() => {
-        expand.focus();
-        fireEvent.keyUp(expand, { key: ' ' });
-      });
+      fireEvent.click(expand);
```

Comment:
`fireEvent` already wraps in `act`. `act` is only needed with `expand.click()` which does not do anything different as far as I can tell.